src/Loader/LASLoader.js

```diff
@@ -47,7 +48,11 @@ class LASLoader {
     }
 
     _parseView(view, options) {
-        const colorDepth = options.colorDepth ?? 16;
+        const colorDepth = options.colorDepth ?? defaultColorEncoding(options.header);
+
+        const forward = (options.in.crs !== options.out.crs) ?
+            proj4(options.in.projDefs, options.out.projDefs).forward :
+            (x => x);
```

Comment:
this is same part with foward, could you factorize ?

Reply:
LASLoader will be called by the worker, so we need to limite the dependances, thus it would be counter-productive.